biome.json

```diff
@@ -86,7 +86,8 @@
       "esm",
       "framegear",
       "playground",
-      "site"
+      "site",
+      "src/**/*.test.tsx"
```

Comment:
We want our test files be nice an clean as well. Please remove this line.
And if there are exception you want to make in some test file, do it at line level.

Or if there is a particular file that is super complex, we can talk about it case by case.